scripts/validate_requirements.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+"""Validate custom wheels package requirements."""
+import json
+from pathlib import Path
+import re
+import subprocess
+import sys
+from typing import Any, Dict, List, Optional, Set
+from urllib import request
+from urllib.error import URLError
+
+from stdlib_list import stdlib_list
+
+PACKAGE_REGEX = re.compile(r"^(?:--.+\s)?([-_\w\d]+).*(?:==|>=|<=|~=|!=|<|>|===)?.*$")
+PIP_REGEX = re.compile(r"^(--.+\s)?([-_\w\d]+.*(?:==|>=|<=|~=|!=|<|>|===)?.*$)")
+SUPPORTED_PYTHON_VERSIONS = ["3.8"]
+STD_LIBS = {version: set(stdlib_list(version)) for version in SUPPORTED_PYTHON_VERSIONS}
+
+
+def main() -> int:
+    """Run main function."""
+    return 0 if validate() else 1
+
+
+def validate() -> bool:
+    """Do the validation."""
+    print("Validating requirements")
+    print()
+    piped_input = sys.stdin.read()
+    input_files = piped_input.split("\n")
+    input_component_files = collect_component_files(input_files)
+    validated_ok = True
+    print("Integrations to validate:", len(input_component_files))
+
+    for fil in sorted(input_component_files):
+        manifest = get_manifest(fil)
+        if not manifest:
+            validated_ok = False
+            continue
+        print()
+        print(f"Validating {manifest['domain']}:")
+        requirements = set(manifest["requirements"])
+        requirements_ok = validate_requirements(requirements)
+        if requirements_ok:
+            print("OK!")
+        validated_ok = requirements_ok
```

Comment:
This should consider the existing validation state too. If that is False the result should be False too.
```suggestion
        validated_ok = validated_ok and requirements_ok
```